frontend/apps/reader/modules/readermenu.lua

```diff
@@ -285,7 +285,7 @@ function ReaderMenu:setUpdateItemTable()
         end
         table.insert(screensaver_sub_item_table, ss_book_settings)
         self.menu_items.screensaver = {
-            text = _("Screensaver"),
+            text = _("Lock Screen"),
```

Comment:
No strong opinion about Screensaver vs Lock screen (they both suck :) it saves and locks nothing for me :))

But our top menu would expect no middle sentence capitalization I think. `Lock screen` rather than `Lock Screen`.
(I see you made a lot of such things _Movie Title Case_.)

Reply:
Screensaver is much less inaccurate though. It behaves like a screensaver while it behaves nothing like a lock screen. If we did change the word `screensaver` I'd rather change it to something like `sleep image`.

Reply:
this is the option you get after `cog symbol > screen > lock screen`. I am sorry @Frenzie but it is a Lock Screen, the screen is powered off and well... locked. There is a sub menu just after that one `cog symbol > screen > lock screen > screensaver` which refers just to the image displayed when the screen is locked. I thought about `wallpaper` but I liked that less so..